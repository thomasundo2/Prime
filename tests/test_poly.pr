int main()
{
    poly x;
    printpoly([(1l, 2l) : 3l]);

    x = [(1l, 2l) : 3l];
    printpoly(x);

    return 0;
}
