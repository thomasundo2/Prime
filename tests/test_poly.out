<(1l, 2l), 3l>
<(1l, 2l), 3l>

LBRACK LPAREN expr COMMA expr RPAREN COLON expr RBRACK {Polylit($3, $5, $8)}
